crates/autopilot/src/arguments.rs

```diff
@@ -148,9 +148,11 @@ pub struct Arguments {
     #[clap(long, env, default_value = "0")]
     pub limit_order_price_factor: f64,
 
-    /// Enable background quoting for limit orders.
-    #[clap(long, env)]
-    pub enable_limit_orders: bool,
+    #[clap(long, env, default_value = "true")]
```

Comment:
Setting `default_value = true` for bool flags means we can only disable it with an environment variable.

Reply:
Does that mean it *has* to become "disable_limit_orders" if I want the default to be to have them enabled?

Reply:
If we want to support disabling and enabling this feature with CLI arguments I would say yes. But it's also not possible to combine CLI arguments and environment variables when you run the binary locally (which is probably the only case where this matters). I personally don't feel very strongly about it and I can see the reason why you would want to make it the default behavior.
Maybe other people also have an opinion on this.

Reply:
I figured out how to allow default = true while also allowing the argument to be disabled. See most recent commit.

Reply:
Neat! 👍 
This changes the default behavior from passing `--flag` (which only allows to set the flag to true) to `--flag [true|false]` which gives us the total control.